src/main/resources/static/javascript/account.js

```diff
@@ -0,0 +1,184 @@
+$(document).ready(function () {
+
+    function postData() {
+
+        var firstName = $("#inputFirstName").val();
+        var lastName = $("#inputLastName").val();
+        var email = $("#inputEmail").val();
+        var birth = $("#inputBirth").val();
+        var phone = $("#inputPhone").val();
+        var address = $("#inputAddress").val();
+        var city = $("#inputCity").val();
+        var zipCode = $("#inputZipCode").val();
+        var state = $("#inputState").val();
+        var country = $("#inputCountry").val();
+
+        var newAccount = {
+            "firstName": firstName,
+            "lastName": lastName,
+            "password": "$2a$10$AIUufK8g6EFhBcumRRV2L.AQNz3Bjp7oDQVFiO5JJMBFZQ6x2/R/2",
+            "dateOfBirth": birth,
+            "mail": email,
+            "phone": phone,
+            "address": address,
+            "city": city,
+            "zipCode": zipCode,
+            "country": country,
+        };
+
+        var JsonAccount = JSON.stringify(newAccount);
+
+        $.ajax({
+            url:"http://localhost:8080/guest/",
+            type:"post",
+            data: JsonAccount,
+            contentType: "application/json",
+            success: function(result) {
+                $("#newAccountModal").hide();
+                $('body').removeClass('modal-open');
+                $('.modal-backdrop').remove();
+                getData();
+                $("#inputFirstName").val("");
+                $("#inputLastName").val("");
+                $("#inputEmail").val("");
+                $("#inputBirth").val("");
+                $("#inputPhone").val("");
+                $("#inputAddress").val("");
+                $("#inputCity").val("");
+                $("#inputZipCode").val("");
+                $("#inputState").val("");
+                $("#inputCountry").val("");
+            }
+        });
+    }
+
+    function updateData() {
+        var firstName = $("#updateFirstName").val();
```

Comment:
dit zou één functie kunnen zijn voor post en update.. get account of wat dan ook als jSON object

Reply:
sowieso zou post en update hetzelfde kunnen zijn, op de request type na... minder copy pasten